src/main/java/frc/robot/commands/operatorcommands/TeleFly.java

```diff
@@ -0,0 +1,33 @@
+package frc.robot.commands.operatorcommands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import java.util.function.Supplier;
+
+import frc.robot.subsystems.flywheel.FlyWheel;
+import frc.robot.subsystems.flywheel.FlyWheelIOVariable;
+
+public class TeleFly extends CommandBase{
+    private final FlyWheel flyWheel;
+    public TeleFly(FlyWheel flyWheel) {
+        this.flyWheel = flyWheel;
+    }
+
+    @Override
+    public void execute() {
+        if(FlyWheelIOVariable.isExtended) {
+            flyWheel.flyVelocity(422, 422);
+        } else if(!FlyWheelIOVariable.isExtended){
+            flyWheel.flyVelocity(422/2, 422/2);
+        }
```

Comment:
We should not depend on static variable from a specific class here. Should all come from `flyWheel` field

Reply:
alright I moved it to the flywheel field